spec/support/test_helpers/provider_user_permissions_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ProviderUserPermissionsHelper
+  def permit_make_decisions(dfe_sign_in_uid: 'DFE_SIGN_IN_UID', provider: nil)
+    FeatureFlag.activate 'provider_make_decisions_restriction'
+
+    provider_user = ProviderUser.find_by_dfe_sign_in_uid dfe_sign_in_uid
+    permissions = if provider
+                    provider_user.provider_permissions.where(provider: provider)
+                  else
+                    provider_user.provider_permissions
+                  end
+
+    permissions.update_all(make_decisions: true)
+  end
+
+  alias_method :and_i_am_permitted_to_make_decisions_for_my_provider, :permit_make_decisions
```

Comment:
The rule of thumb for helpers is to share implementations, not steps, so I think the project-standard way to do this would be to implement `and_i_am_permitted_to...` in each spec that uses it, and call out to `permit_make_decisions` each time ([Rule 3](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/docs/testing-styleguide.md#rules))